example/annotation/app.js

```diff
@@ -58,12 +58,9 @@ document.addEventListener('DOMContentLoaded', function() {
             //         wavesurfer.getDuration()
             //     )
             // );
-            wavesurfer.util
-                .ajax({
-                    responseType: 'json',
-                    url: 'annotations.json'
-                })
-                .on('success', function(data) {
+            fetch('/example/annotation/annotations.json')
```

Comment:
no need to use an absolute url here, i'd prefer to use the prev url format (unless it somehow doesn't work with such a relative url but i doubt it)

Reply:
fixed